api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -98,35 +98,6 @@ public static CorrelationContextManager getCorrelationContextManager() {
     return getInstance().contextManager;
   }
 
-  /**
-   * Returns a {@link ContextPropagators} object, which can be used to access the set of registered
-   * propagators for each supported format.
-   *
-   * @return registered propagators container, defaulting to a {@link ContextPropagators} object
-   *     with {@link HttpTraceContext} registered.
-   * @throws IllegalStateException if a specified manager (via system properties) could not be
-   *     found.
-   * @since 0.3.0
-   */
-  public static ContextPropagators getPropagators() {
```

Comment:
So.. with this change, the instrumentors no longer get propagators from the central OpenTelemetry location. 
What's the rationale? 

Having one place for instrumentors seems like a good thing, IMO.

Reply:
The rationale is that context-prop logic is rather independent piece and should be able to be used even without the OTel API.

That being said, I still slightly prefer the current design we have at this moment (just like you ;) )

Reply:
You could keep the method in the OpenTelemetry class, even if there's another source for it, yes?

Reply:
This part, yes, we could do that. The question is whether we also want the setter (guess not?).